Tests/WebAuthnTests/Utils/TestModels/TestAuthData.swift

```diff
@@ -62,7 +62,7 @@ struct TestAuthDataBuilder {
     func validMock() -> Self {
         self
             .rpIDHash(fromRpID: "example.com")
-            .flags(0b01000101)
+            .flags(0b11000101)
```

Comment:
Since extension data was always provided, but the flag was unset, none of the tests detected this.